packages/Panel/stories/tests/PanelFooterSticky.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Panel from "../../src";
+import { TextLine, Nav } from "../helpers";
+
+export default function PanelFooterSticky() {
```

Comment:
What do you think about having all the `tests` stories exported from a single file? 

I think it's important for `examples` stories to be isolated for being able to clearly link to them in GitHub from storybook, but the test stories are pretty short and could reasonably fit in one file making maintenance a little easier?

Reply:
True, i can condense them to one file